acl/README.md

```diff
@@ -13,3 +13,47 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+# Testing acl on Linux
+
+## Build, Install and Test the acl core plan
+
+### Build and Install
+
+Open a terminal at the root directory of the core-plans repo.  Build and install the package:
+
+```bash
+hab pkg build acl
+source ./results/last_build.env
+```
+
+### Test
+
+In a different terminal test the package:
+
+```bash
+source ./results/last_build.env
```

Comment:
It's unnecessary to tell someone to open a terminal.

The test section should read as follows:


```bash
hab pkg build acl
source ./results/last_build.env
hab studio run "./acl/tests/test.sh ${pkg_ident}"
```

Reply:
Amended.